docs/reference/wm8960.rst

```diff
@@ -0,0 +1,420 @@
+.. _wm8960:
```

Comment:
please move this to `docs/library/wm8960.rst`

Reply:
Done.